x-pack/plugins/apm/public/components/app/error_group_overview/index.tsx

```diff
@@ -94,27 +100,53 @@ export function ErrorGroupOverview() {
               },
             }
           ).then((response) => {
+            const currentPageErrorGroupMainStatistics = orderBy(
+              response.errorGroups,
+              sortField,
+              sortDirection
+            ).slice(page * pageSize, (page + 1) * pageSize);
+
             return {
               // Everytime the main statistics is refetched, updates the requestId making the comparison API to be refetched.
               requestId: uuid(),
               errorGroupMainStatistics: response.errorGroups,
+              currentPageErrorGroupMainStatistics,
```

Comment:
We only need the groupId's right?

```suggestion
          const currentPageGroupIds = orderBy(
            response.errorGroups,
            sortField,
            sortDirection
          )
            .slice(page * pageSize, (page + 1) * pageSize)
            .map(({ groupId }) => groupId)
            .sort();

            return {
              // Everytime the main statistics is refetched, updates the requestId making the comparison API to be refetched.
              requestId: uuid(),
              errorGroupMainStatistics: response.errorGroups,
              currentPageGroupIds,
```

Reply:
Btw. why are we sorting the group ids the second time?